### PR TITLE
Add tRO Server details

### PR DESCRIPTION
**Pull request by bouroo**
Add tRO Server details that try extract from https://ro.gnjoy.in.th/ game client

### CODE REVIEW

File: tables/servers.txt [hunk 1]
```diff
@@ -289,37 +289,43 @@ recvpackets recvpackets.txt
...
+secureLogin 0
+secureLogin_type 0
+secureLogin_requestCode
+secureLogin_account 0
 serverEncoding Thai
-addTableFolders tRO
-charBlockSize 146
 gameGuard 0
+addTableFolders tRO
+charBlockSize 144
```
Comment (sctnightcore) [inline tables/servers.txt:325]: I think `charBlockSize` isnot 144 ?
it use with new ui.

File: tables/servers.txt [hunk 2]
```diff
@@ -289,37 +289,43 @@ recvpackets recvpackets.txt
...
```
Comment (sctnightcore) [inline tables/servers.txt:306]: I think `charBlockSize` isnot 144 ?
it use with new ui.

Comment (bouroo) [reply tables/servers.txt]: Thx I'll change to correct value.

**Comment by bouroo** [comment]
So I've changed master version, char block size, roll back recvpackets.
After some test. I can logged into server, character selectable and make a few move in map.

File: tables/servers.txt (hunk 1 above)
Comment (bouroo) [reply tables/servers.txt]: Changed in next commit

**Comment by bouroo** [comment]
tRO split 3 login servers @ 30/05/2020

**Comment by compaqname** [comment]
> tRO split 3 login servers @ 30/05/2020

image

**Comment by compaqname** [comment]
Not move in map . . and disconnect pls.

File: control/config.txt [hunk 3]
```diff
@@ -804,4 +804,4 @@ logToFile_Errors
 logToFile_Messages
 logToFile_Warnings
 history_max 50
-gameGuard 0
+gameGuard 0
```
Comment (sctnightcore) [inline control/config.txt:807]: del it .

**Comment by sctnightcore** [comment]
> Not move in map . . and disconnect pls.
it dont work with EAC

**Comment by bouroo** [comment]
> Not move in map . . and disconnect pls.

So we've lot to do with EAC

**Comment by compaqname** [comment]
> > Not move in map . . and disconnect pls.
> 
> So we've lot to do with EAC

Now not opk can't use with tRo ?

**Comment by bouroo** [comment]
> > > Not move in map . . and disconnect pls.
> > 
> > 
> > So we've lot to do with EAC
> 
> Now not opk can't use with tRo ?

Now OPK not bottable in servers that protected by EAC.
That we need more people to deal with this problem.

**Comment by tummadee555** [comment]
Can Xkore1 get a tro?

**Comment by bouroo** [comment]
change base to tRO-Update.
And add Sakray server details

**Comment by zuroba** [comment]
@unknown-item 
can do?

**Comment by nopth** [comment]
The map has changed, such as prt_fild08b, which then stops the connection. From before, it will be prt_fild08.

**Comment by ThememoriesIII** [comment]
1.openkore connect to mapserver a character stuck and dont move
2.after connect to mappserver  mappsend timerout and disconnect
2.another player around character can move. i open debug in config.txt and see character_move value 035F change to 000D and lenght value 5 change to 13,

**Pull request closed without merging**